src/tools/listFileIssuesTool.ts

```diff
@@ -8,6 +8,8 @@ const rules = `
   Common use cases: 
   - When the user asks for the issues for a specific file
   - When the user is in a file context and asks for the issues
+  - When the user asks for the quality analysis of a specific file
```

Comment:
quality analysis includes duplication and complexity too, so I don'y this we should mention this here. it should use the file with analysis